onnxruntime/core/optimizer/attention_fusion_helper.h

```diff
@@ -550,6 +550,7 @@ bool MatchUnidirMaskSubgraph(const Graph& graph, const Node& add_node, MatchUnid
 struct AttentionMaskNodes {
   const Node* softmax;
   bool has_input_mask;  // When it is false, the following nodes will be NULL.
+  float mask_filter_value = -10000.0f;
```

Comment:
nit: define a constexpr variable for -10000.0f (maybe put in attention_common.h), and reference it in every places.

Reply:
will do in future PR